lib/Helpers/ASM.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * This helper builds the request body for a /mail/send API call.
+ *
+ * PHP version 5.6, 7
+ *
+ * @author    Elmer Thomas <dx@sendgrid.com>
+ * @copyright 2017 SendGrid
+ * @license   https://opensource.org/licenses/MIT The MIT License
+ * @version   GIT: <git_id>
+ * @link      http://packagist.org/packages/sendgrid/sendgrid
+ */
+
+namespace Helpers;
```

Comment:
I left out the Model directory for now to see what it looks like, and to get your take. I think I'm liking the more succinct convention. Give it a gander.